### ISSUE TITLE
Improve empty window of FS to be more productive

### ISSUE THREAD

**Issue by MakeshVineeth** [adirh3/Fluent-Search#144]
When we have no processes opened and we try to launch FS without searching anything, it will display an empty window like this:

![2021-06-21](https://user-images.githubusercontent.com/19371684/122708471-e2084280-d279-11eb-88f5-c1f1468307d4.png)

But it is looking boring and it can be made more productive by including some useful information like the following:

- Display some useful tips at the **bottom** corner of this empty window. Those tips can refer to the usage of FS like Tips to get most out of the Fluent Search.
- Include some useful insights like Recently Typed Searches, recently used Plugins or recently accessed files through FS. (Also, an option to disable them all for privacy concerned users.)

Basically, something similar to Windows Search. It will be more productive that way.

**Comment by edenexposito** [comment]
Show Latest modified files and folders will be a GREAT improvement!

**Comment by Nomad-0** [comment]
yeah, this can be done when the window style is set to _full window_. Recent files can also be shown

**Comment by adirh3** [comment]
This is planned for next version

**Comment by MakeshVineeth** [comment]
These are the combined suggestions for the new home screen feature. Posted by me, HPR and Najah on Discord. Just posting it here altogether for a quick overview.

- UI can be improved, right now, it looks bit too plain.
- Some of the results under Recent has Edit in the Right-Click Menu but it does nothing as of now. (Mainly Bookmarks)
- Microsoft Edge has no icon, I guess it's the old one that Microsoft didn't remove as @Adir mentioned previously. (See attached image)
- Results under Recent should have Delete in their Context Menu to delist them from Recent view.
- Unpinned custom tag folders will reappear again after restart.
- Add an option to pin tag folders in the Right Panel view of the searched custom tag. (Or through context menu that appears on right-clicking of tag text, See [**this**](https://discord.com/channels/740636963396124714/854129579849023498/856827475640188938) for more info.
- Clicking on a tag folder should divert to their respective search results.
- I think 2 seconds for drag and drop is a bit too much
- It would be great if there are hotkeys for pinned results
- The Unpin action doesn't highlight like others: https://github.com/adirh3/Fluent-Search/issues/180#issue-931739125
- A UI indicator when reordering the pinned results
- A divider or something that divides the home screen and search bar
- Types shouldn't show on home screen when minimal user interface is turned on.
- Alternatively, there could be more distinction by grouping per category.

![image](https://user-images.githubusercontent.com/19371684/123831715-3b2f4080-d922-11eb-817f-3683c2420d7d.png)

**Comment by MakeshVineeth** [comment]
@adirh3 Home Screen related issues found in v0.9.88.72

- **Uninstall App** operation of a UWP app doesn't seem to work, tried through Home Screen.
- Apps uninstalled through Start Menu, are not being updated in FS real-time. They still show show-up on both Home Screen and Search window even after uninstall.
- Search Results from custom plugins are automatically being saved on Home Screen when one of the operations are used. While this feature is great but it seems to break some plugins completely i.e., next time whenever we use a plugin like Wikipedia, only the saved results from Home Screen will show up. **Workaround:** Have to remove those search results from Home Screen every time.
- Sometimes, suggestions are spaced out in Home Screen as shown in below picture. **Workaround:** After 2 or 3 restarts, it automatically got fixed.

![FS_Bug (Small)](https://user-images.githubusercontent.com/19371684/125306793-22288580-e34d-11eb-9269-2542c6b40708.png)

**Comment by MakeshVineeth** [comment]
Some more suggestions for Home Screen window: (This has been posted on Discord, but posting it here due to relevance of this thread)

Since FS is all about productivity, I feel it is prudent to occasionally display tips/tricks on Home Screen. In a big and flexible software like this, often we may forget a thing or two. It would be awesome if FS can occasionally remind us that we're doing something wrong or doing it the hard way, there is a better and faster way to do your task using FS. The hints can also be added to bottom of the Home Screen. It should start with **Did you know?** and a hint text.

As an alternative to above suggestion, I like when the watermark "Search anything" text keeps changing and with some nice typing animations. That way, it can also give some hints on what all things can be searched. In my view, it should not be static/fixed text. Having it changed to random hints at set time intervals can make Home Screen look even richer. In my view, it should not be static/fixed text. Having it changed to random hints at set time intervals can make Home Screen look even richer.

**Comment by adirh3** [comment]
@MakeshVineeth can you review it and see if anything is missing from the original request?
About the hints/recommendations, if you can open a new issue for it with more info it would be great.
Thanks!

**Comment by MakeshVineeth** [comment]
Closing this issue, as most of the suggestions were added in the latest FS Versions. Will open a separate issue for hints on Home Screen. Thanks for this great feature @adirh3 :)
